### PR TITLE
Add 2 more permissions

### PR DESCRIPTION
**Pull request by greatbacon**
Branch name was a little ambitious lol

Anyway, this adds 2 more checks. 

1 is a standard permission check for `Ephemera`

The other is for DeletePersona, which ran up against a corner case in tests which I'll detail below.

### CODE REVIEW

File: src/lib/server/services.test.ts [hunk 1]
```diff
@@ -141,23 +141,27 @@ for (const service of services.values()) {
 
 			if (
 				[
+					'DeletePersona', //This event is outside the policy system and requires mismatched actor/persona_id to fail
```
Comment (greatbacon) [inline src/lib/server/services.test.ts:144]: So, the way the code in this test works, it sets up the `unauthorizedPersona`, but then goes on to use the same persona_id as `actor` and `persona_id` for event params. This however is explicitly the expected behavior for this event, so it fails to fail (lol).

Anyway, I wasn't quite sure about how to go about hacking this edge case, so I just left it in the list for now.

File: src/lib/vocab/persona/personaServices.test.ts [hunk 2]
```diff
@@ -120,5 +121,38 @@ test__personaService('delete a persona and properly clean up', async ({db, rando
...
+			params: {actor: persona1.persona_id, persona_id: persona2.persona_id},
+		}),
+	);
+});
+
+test__personaService(
+	'actors can delete other personas if in the admin community',
+	async ({db, random}) => {
+		const account = await random.account();
+		const {persona} = await random.persona(account);
```
Comment (ryanatkn) [inline src/lib/vocab/persona/personaServices.test.ts:141]:
```suggestion
		const {persona} = await random.persona();
```

File: src/lib/vocab/persona/personaServices.ts [hunk 3]
```diff
@@ -129,9 +129,10 @@ export const CreateAccountPersonaService: ServiceByName['CreateAccountPersona']
 export const DeletePersonaService: ServiceByName['DeletePersona'] = {
 	event: DeletePersona,
 	perform: async ({repos, params}) => {
-		const {persona_id} = params;
+		const {actor, persona_id} = params;
 
 		// first check if deleting the persona is allowed
+		//TODO extract to it's own policy helper?
```
Comment (ryanatkn) [inline src/lib/vocab/persona/personaServices.ts:135]: good todo, wondering if we want a way to refer to these, like "system personas"? so then they helper may be `isSystemPersona`?

**Review by ryanatkn — APPROVED** [review]
excellent, wrap up incoming!

File: src/lib/server/services.test.ts (hunk 1 above)
Comment (ryanatkn) [reply src/lib/server/services.test.ts:144]: lol hm I'll think on it, no harm in leaving for now, a hardcoded hack may be fine
